data/update/mysql/dbupdate-5.3.0-to-5.4.0.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE ezsearch_return_count;
```

Comment:
One last thing, platform users will need to drop this during an upgrade, so move this and the other one to`dbupdate-6.10.0-to-6.11.0.sql` 

And add inline comment like `--eZ Publish users who has upgraded will get warning that table was missing, this is fine as you'll already removed it when upgrading to 5.4`